src/test/java/com/epam/tc/hw3/bonus/fluent/tests/FluentTestElementsPage.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.tc.hw3.bonus.fluent.tests;
+
+import com.epam.tc.hw3.bonus.fluent.pages.FluentDifferentElementsPageObject;
+import com.epam.tc.hw3.bonus.fluent.pages.FluentHomePageObject;
+import com.epam.tc.hw3.tests.PagesUITest;
+import java.util.Map;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class FluentTestElementsPage extends PagesUITest {
+
+    private FluentHomePageObject homePage;
+    private FluentDifferentElementsPageObject elementsPage;
+
+    @BeforeClass
+    void navigateToHomePage() {
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+        homePage = new FluentHomePageObject(webDriver);
+    }
+
+    @DataProvider(name = "loginData")
+    Object[][] loginData() {
```

Comment:
It's better to move all data providers to a separate class

Reply:
Done